exo-run/features/run.feature

```diff
@@ -54,3 +54,13 @@ Feature: running Exosphere applications
   Scenario: a service crashes during startup
     When trying to run the "crashing-service" application
     Then it prints "crasher exited with code 1" in the terminal
+
+
+  Scenario: hiding logs from certain service and dependency
+    Given a running "silenced-running" application
+    Then my machine is running the services:
+      | NAME  |
+      | web   |
+      | users |
```

Comment:
You might also have to add a step before line 65 that checks that 'all services online' is printed. That way we're not checking for the absence of text before the application has been fully booted up.